www/index.md

```diff
@@ -2,7 +2,8 @@
 title: Home
 pageTitle: 'About <abbr title="Design Tokens Community Group">DTCG</abbr>'
 date: 2020-05-24
-permalink: /
+layout: layouts/page.njk
```

Comment:
There was a change to the `pages/` folder, now it’s no longer “magic” and things just live where they live (`/index.md` → `/index.html` so it’s now a 1:1 mapping with no indirection). We could get it back with more config, but I personally prefer explicitness, and less config.